OracleGoldenGate/createDB.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
```

Comment:
So I couldn't find an elegant way to integrate the required changes for GoldenGate upon database creation to enable archive logging,  supplemental logging and create the necessary GoldenGate user with the necessary privileges.   I had considered doing it after instance creation but I'd have to somehow wait until the createDB is complete before I run the additional steps outlined below.  I'm open to suggestions if you have any thoughts...

I added these to the bottom of that the createDB.sh:   

`# Modified by Rick Michaud for GoldenGate purposes
# Setting up database logging for GoldenGate
echo "Configuring database logging for GoldenGate..."
sqlplus / as sysdba << EOF
   set echo on
   set timing on
   SHUTDOWN IMMEDIATE;
   STARTUP MOUNT;
   ALTER DATABASE ARCHIVELOG;
   ALTER DATABASE OPEN;
   ALTER SYSTEM SWITCH LOGFILE;
   ALTER SESSION SET CONTAINER=$ORACLE_PDB;
   ALTER PLUGGABLE DATABASE ADD SUPPLEMENTAL LOG DATA;
   exit;
EOF

# Create GoldenGate User for Source Extract
echo "Creating GoldenGate user and granting appropriate privileges..."
sqlplus / as sysdba << EOF
   set echo on
   set timing on
   CREATE USER C##ggadmin IDENTIFIED BY ggadmin DEFAULT TABLESPACE USERS TEMPORARY TABLESPACE TEMP;
   EXEC dbms_goldengate_auth.grant_admin_privilege('C##ggadmin', CONTAINER=>'ALL');
   GRANT DBA TO C##ggadmin CONTAINER=ALL;
   exit;
EOF
`